www/js/insert-slide.js

```diff
@@ -0,0 +1,188 @@
+const sanitizeHtml = require('sanitize-html');
+const electron = require('electron');
+
+const analytics = electron.remote.getGlobal('analytics');
+const tingle = require('./vendor/tingle');
+const strings = require('./strings');
+
+// allowed html tags inside announcement
+const allowedTags = [
+  'b',
+  'i',
+  'em',
+  'u',
+  'pre',
+  'strong',
+  'div',
+  'code',
+  'br',
+  'p',
+  'ul',
+  'li',
+  'ol',
+];
+
+/**
+ * boolean to check what modal page is asctive
+ * false = dhan guru slide page
+ * true = announcement page
+ * default to false b/c dhan guru slide page is default
+ */
+let isAnnouncementTab = false;
+const modal = new tingle.Modal({
+  footer: true,
+  stickyFooter: false,
+  closeMethods: ['overlay', 'button', 'escape'],
+  onClose() {},
+  beforeClose() {
+    return true; // close the modal
+  },
+});
+
+// slide modal  body
+
+// title
+const slideHeader = '<h1 class = "modalTitle"> <center> Insert Slide </center></h1>';
+// section title
+const slideText = '<h2 class ="dhanguru">Dhan Guru:</h2>';
+// button group
+const buttons = '<center><div class="btn-group" id = "btn-group">'.concat(
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance something to consider. performance should be negligible in this case but your call if you want to change it.